src/app/cube/new-home/learning-object-info/learning-objects/learning-objects.component.html

```diff
@@ -1 +1,51 @@
-<p>learning-objects works!</p>
+<div class="wrapper">
+    <div class="learning-object-info">
+        <h1>Modular self-contained cyber curricula</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
+            nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in 
+            reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
+            Excepteur sint occaecat cupidatat non proident, 
+            sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </p>
+    </div>
+
+    <div class="featured-object">
+        <div class="featured-text">
+            <p *ngIf="featuredObject?.levels" class="levels">
+               <b>{{ displayFeaturedObjectLevels().toUpperCase() }}</b>
+            </p>
+            <h3 *ngIf="featuredObject?.author">
+                <a [routerLink]="[
+                '/details',
+                featuredObject.author.username,
+                featuredObject.cuid
+                ]">
+                {{ featuredObject?.name }}
+                </a>
+            </h3>
+            <p *ngIf="featuredObject?.description" [innerHTML]="displayDescription()">
+            </p>
+            <p class="outcomeText">
+                Students will be able to...
+            </p>
+            <ul>
+                <li *ngFor="let outcome of featuredObject?.outcomes">
+                    {{ capitalize(outcome.verb) }} {{ outcome.text }}
```

Comment:
```suggestion
                    {{ outcome.verb | titlecase }} {{ outcome.text }}
```
You can use the titlecase pipe here and you'll get what ur looking for. You won't need the capitalize() function. [here's](https://angular.io/api/common/TitleCasePipe) the docs

Reply:
That's pretty cool, would have saved me a lot of time had I known this 🥲